lib/cinch/plugins/wordgames/game.rb

```diff
@@ -1,19 +1,19 @@
 require 'cinch'
 
 require_relative 'dictionary.rb'
-require_relative 'word.rb'
+require_relative 'solution.rb'
 require_relative 'response.rb'
 
 module Cinch::Plugins
   class Game
-    attr_reader :number_of_guesses, :word, :lower_bound, :upper_bound
+    attr_reader :number_of_guesses, :lower_bound, :upper_bound, :solution
     Blank_str = "__"
 
     def initialize(dictionary, ref_dict)
       @dict = dictionary
       @ref_dict = ref_dict
-      @word = Word.new(@dict.random_word)
-      puts "Word is #{@word}"
+      @solution = Solution.new(@dict.random_word)
```

Comment:
I don't agree with the class rename.. you could have just done `@solution = Word.new...` afterall, it's actually a Word class (other words can use it, not just solutions). I'm also not fan of the word "solution" as it implies more than this simply being a word.. how about  `target_word` or `chosen_word` (use that)